dkg/sync/client.go

```diff
@@ -85,10 +85,10 @@ func (c *Client) Run(ctx context.Context) error {
 		relayBroke, connBroke, err := c.sendMsgs(ctx, stream)
 		c.clearConnected()
 		if relayBroke {
-			log.Debug(ctx, "Relay connection dropped, reconnecting")
+			log.Warn(ctx, "Relay connection dropped, reconnecting", nil, z.Str("peer", p2p.PeerName(stream.Conn().RemotePeer())))
```

Comment:
so relay connections breaking is expected (it happens every 2min), we mostly ignore relay errors. Please revert this change.